elasticsearch/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/ose-base:ubi7
+FROM ubi7:latest
```

Comment:
same question here about what does ART/CI use?

Reply:
openshift/ose-base:ubi7

Reply:
As ART answered above the images are the same.